ui/tests/acceptance/deployment-resource-list-test.ts

```diff
@@ -32,7 +32,7 @@ module('Acceptance | deployment resource list', function (hooks) {
     let deployment = this.server.create('deployment', 'random', { application, sequence: 1 });
     this.server.create('status-report', 'ready', { application, target: deployment });
 
-    await visit(`/default/${project.name}/app/${application.name}/deployments/${deployment.sequence}`);
+    await visit(`/default/${project.name}/app/${application.name}/deployments/${deployment.sequence}]`);
```

Comment:
```suggestion
    await visit(`/default/${project.name}/app/${application.name}/deployments/${deployment.sequence}`);
```

question: not needed here?

Reply:
Good eyes! That’ll teach me to make a change like this last thing at night.